ccd-definition/CaseEventToFields/CareSupervision/manageOrders/3a_hearingDetailsSection.json

```diff
@@ -62,5 +62,44 @@
     "PageFieldDisplayOrder": 4,
     "FieldShowCondition": "orderTempQuestions.hearingDetails=\"YES\" AND manageOrdersApprovedAtHearing=\"Yes\"",
     "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "03/06/2021",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersShouldAllowLinkingApplication",
```

Comment:
without having to declare an extra yes/no, how about implement a logic in the prepopulator similar to CloseCaseBlockPrePopulator 

   return Map.of("orderTempQuestions", orderTempQuestions.toBuilder().closeCase("NO").build());